src/types.ts

```diff
@@ -58,21 +58,40 @@ export interface VolumeChannelDisplayOptions {
    };
  */
 export interface VolumeDisplayOptions {
-  channels: VolumeChannelDisplayOptions[];
-  density: number;
-  translation: [number, number, number];
-  rotation: [number, number, number];
-  maskChannelIndex: number;
-  maskAlpha: number;
-  clipBounds: [number, number, number, number, number, number];
-  scale: [number, number, number];
-  maxProjection: boolean;
-  renderMode: number;
-  shadingMethod: number;
-  gamma: [number, number, number];
-  primaryRayStepSize: number;
-  secondaryRayStepSize: number;
+  channels?: VolumeChannelDisplayOptions[];
+  density?: number;
+  translation?: [number, number, number];
+  rotation?: [number, number, number];
+  maskChannelIndex?: number;
+  maskAlpha?: number;
+  clipBounds?: [number, number, number, number, number, number];
+  scale?: [number, number, number];
+  maxProjection?: boolean;
+  renderMode?: number;
+  shadingMethod?: number;
+  gamma?: [number, number, number];
+  primaryRayStepSize?: number;
+  secondaryRayStepSize?: number;
 }
 
 export const isOrthographicCamera = (def: Camera): def is OrthographicCamera =>
   def && (def as OrthographicCamera).isOrthographicCamera;
+
+export interface ChannelGuiOptions {
+  colorD: [number, number, number];
+  colorS: [number, number, number];
+  colorE: [number, number, number];
+  window: number;
+  level: number;
+  glossiness: number;
+  isovalue: number;
+  isosurface: boolean;
+  enabled: boolean,
+  autoIJ: (channelNum: number) => void;
+  auto0: (channelNum: number) => void;
+  bestFit: (channelNum: number) => void;
+  pct50_98: (channelNum: number) => void;
```

Comment:
this `pct50_98` needs a linter disable comment.. also, if this `ChannelGuiOptions` is only used in the public/ demo viewer, then it doesn't need to be under `src/`